pkg/virt-handler/isolation/detector.go

```diff
@@ -50,6 +50,10 @@ type PodIsolationDetector interface {
 
 	DetectForSocket(vm *v1.VirtualMachineInstance, socket string) (IsolationResult, error)
 
+	// Detects if there is a Pod on the host for given vm
+	// Returns error otherwise
+	DetectPod(vmi *v1.VirtualMachineInstance) error
```

Comment:
Shouldn't this return the `socketPodDir`, error otherwise?
Otherwise I would tend to switch to something like:
`DoesPodExists(vmi *v1.VirtualMachineInstance) (bool, error)`

Reply:
The `socketPodDir` is unused. In fact `FindPodDirOnHost` is now only used once, here.
I thought about `(bool, error)` but I am not sure if there is any value. For now, I like that it preserves the behavior and I don't need to add additional coverage.

I am open to suggestions for names but I do reserve to keep the logic in this PR as close as it was and only follow up with improvements to the logic once we cover this with more unit tests.

Reply:
It's ok to me to leave the things as you proposed, with the promise that `FindPodDirOnHost` will be refactored in the future to drop the `socketPodDir`, if is not needed at all.
IOW my only concern is that we will forget that `FindPodDirOnHost` is returning an unused value.